mayastor/Cargo.toml

```diff
@@ -83,7 +83,7 @@ url = "2.1"
 smol = "1.0.0"
 dns-lookup = "1.0.4"
 ipnetwork = "0.17.0"
-bollard = "0.8.0"
+bollard = "0.10.0"
```

Comment:
This and `ipnetwork` can be removed as it's only needed for the tests, which will pick them up via `composer`.